libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1826,10 +1830,10 @@ std::string YulUtilFunctions::clearStorageRangeFunction(Type const& _type)
 
 	return m_functionCollector.createFunction(functionName, [&]() {
 		return Whiskers(R"(
-			function <functionName>(start, end) {
-				for {} lt(start, end) { start := add(start, <increment>) }
+			function <functionName>(start, slotCount) {
```

Comment:
```suggestion
			function <functionName>(startSlot, slotCount) {
```

And `start` -> `startSlot` in the docstring as well.